src/container/srv_layout.c

```diff
@@ -49,10 +49,77 @@ RDB_STRING_KEY(ds_cont_prop_, roots);
 RDB_STRING_KEY(ds_cont_prop_, ec_cell_sz);
 RDB_STRING_KEY(ds_cont_prop_, ec_pda);
 RDB_STRING_KEY(ds_cont_prop_, rp_pda);
+RDB_STRING_KEY(ds_cont_prop_, cont_global_version);
 
 /* dummy value for container roots, avoid malloc on demand */
 static struct daos_prop_co_roots dummy_roots;
 
+/** default properties, should cover all optional container properties */
+struct daos_prop_entry cont_prop_entries_default_v0[CONT_PROP_NUM_V0] = {
```

Comment:
I wouldn't block on this, but I wonder if there is a more elegant way to do this than duplicating these. With the ACL stuff below, it seems like we do some extra work to handle versions of this prop that hasn't actually changed between versions.

Reply:
I agreed this is not perfect, i might modify they share same ACL ptr.

Reply:
I wonder if maybe it would work to have a base set that represents the latest/greatest, and then have a v0 structure that includes only values that have changed. And perhaps a list of prop names we expect each version to have?